azure-keyvault/azure/keyvault/__init__.py

```diff
@@ -1,19 +1,27 @@
-#---------------------------------------------------------------------------------------------
+# coding=utf-8
+# --------------------------------------------------------------------------
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for license information.
-#---------------------------------------------------------------------------------------------
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
 
-from azure.keyvault import http_bearer_challenge_cache as HttpBearerChallengeCache
-from .http_bearer_challenge import HttpBearerChallenge
-from .key_vault_client import KeyVaultClient
-from .key_vault_authentication import KeyVaultAuthentication
-from azure.keyvault.generated.version import VERSION
+from .custom.key_vault_client import CustomKeyVaultClient as KeyVaultClient
+from .custom.key_vault_id import KeyVaultId
+from .custom import http_bearer_challenge_cache as HttpBearerChallengeCache
+from .custom.http_bearer_challenge import HttpBearerChallenge
+from .custom.key_vault_authentication import KeyVaultAuthentication, KeyVaultAuthBase
+from .version import VERSION
 
-__all__ = [
-    'HttpBearerChallengeCache',
-    'HttpBearerChallenge',
-    'KeyVaultClient',
-    'KeyVaultAuthentication'
-]
+__all__ = ['KeyVaultClient',
+           'KeyVaultId',
+           'HttpBearerChallengeCache',
+           'HttpBearerChallenge',
+           'KeyVaultAuthentication',
+           'KeyVaultAuthBase']
```

Comment:
it's used in the tests and I though it had value allowing people more control in overriding authentication.  If you're really passionate about it I can remove it